### DISCUSSION TITLE
Yarış tahminleri sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#245] [Zeka Sorusu]
Alper, Burçin, Can, Derin ve Erkut 400 metre engelli yarışına katılacaklardır. Yarış öncesi yürüttükleri tahminler şunlardır.

Alper: Burçin Candan iki sıra üstte olacaktır
Burçin: Ben üçüncü olacağım
Can: Derin birinci olacak
Derin: Burçin ikinci olacak
Erkut: Can Alperden üç sıra aşağıda olacak
(Birinci en üst sıradadır)
Yarıştan sonra bu tahminlerin sadece birinin doğru çıktığı görüldü. Doğru tahmini yapan yarışmada birinci olan kişidir. Bu kimdir?

**Comment by YagizEfeZ** [comment]
Yarıştan sonra sadece bir tahminin doğru olduğu söyleniyor bu durumu göz önünde bulundurarak doğru tahmini yapan kişi yani birinci olan kişi Can'dır Çünkü Can'ın tahmini Derin'in birinci olacağını doğru tahmin etmektedir ve sadece bir tahminin doğru olduğu belirtilmişti

Dolayısıyla doğru tahmini yapan ve yarışta birinci olan kişi Can'dır.
